graphics/mesa/system_server.te

```diff
@@ -3,3 +3,4 @@ allow system_server platform_app:file { read write };
 allow system_server priv_app:file { read write };
 allow system_server gpu_device:dir r_dir_perms;
 allow system_server sysfs_app_readable:file r_file_perms;
+allow system_server self:process execmem;
```

Comment:
if you already added "PRODUCT_REQUIRES_INSECURE_EXECMEM_FOR_SWIFTSHADER := true", then this line is not needed.